src/lib/services/access-service.ts

```diff
@@ -244,14 +244,14 @@ export class AccessService {
      * Provided a project, project permissions will be checked against that project.
      * Provided an environment, environment permissions will be checked against that environment (and project).
      */
-    async permissionsMatrixForUser(
+    async getAccessOverviewForUser(
```

Comment:
This change breaks Enterprise temporarily. Should be followed up with the Enterprise PR fixing it.